src/appleseed/renderer/kernel/rendering/pixelrendererbase.cpp

```diff
@@ -51,8 +52,9 @@ namespace renderer
 // PixelRendererBase class implementation.
 //
 
-const uint8 InvalidSampleHint = 1;
-const uint8 CorrectSampleHint = 2;
+const uint8 NoState = 0;
```

Comment:
NoState? Did you mean NoSample?

Reply:
The pixel can be both unsampled or unfinished. In my case, the pixel was sampled but pixel_end was never called.